tests/net/mld/src/main.c

```diff
@@ -327,6 +327,9 @@ static void send_query(struct net_if *iface)
 	/* router alert opt + icmpv6 reserved space + mldv2 mcast record */
 	pkt = net_pkt_alloc_with_buffer(iface, 144, AF_INET6,
 					IPPROTO_ICMPV6, K_FOREVER);
+	if (!pkt) {
```

Comment:
Maybe this should be zassert_true(pkt != NULL, ...) after all?

Reply:
Could be assert too of course. I fixed many of these pkt checks, which Coverity reported, like this. Currently it is not really possible to return from that function with null pkt.

Reply:
> I fixed many of these pkt checks, which Coverity reported, like this. Currently it is not really possible to return from that function with null pkt.

Yeah, at least those had `return NULL`, which is hopefully checked by caller. But here it's potential silent ignoring of test error.